xstream/src/test/com/thoughtworks/xstream/io/naming/NoNameCoderTest.java

```diff
@@ -0,0 +1,27 @@
+package com.thoughtworks.xstream.io.naming;
+
+import com.thoughtworks.xstream.io.naming.NoNameCoder;
+import junit.framework.TestCase;
+
+public class NoNameCoderTest extends TestCase {
+
+  public void testDecodeAttribute() {
+    final NoNameCoder noNameCoder = new NoNameCoder();
+    assertEquals("   ", noNameCoder.decodeAttribute("   "));
+  }
+
+  public void testDecodeNodeInput() {
+    final NoNameCoder noNameCoder = new NoNameCoder();
+    assertEquals("   ", noNameCoder.decodeNode("   "));
+  }
+
+  public void testEncodeAttribute() {
+    final NoNameCoder noNameCoder = new NoNameCoder();
+    assertEquals("   ", noNameCoder.encodeAttribute("   "));
```

Comment:
Choose the encode/decode pairs in a way to show that x = encode(y); y == decode(x)

Reply:
Fixed.